pythran/toolchain.py

```diff
@@ -18,9 +18,10 @@
 from pythran.syntax import check_specs
 import pythran.frontend as frontend
 
-from numpy import get_include
+from numpy.distutils.core import setup
```

Comment:
On Fri, May 22, 2015 at 07:10:23AM -0700, pbrunet wrote:
> > @@ -18,9 +18,10 @@
> >  from pythran.syntax import check_specs
> >  import pythran.frontend as frontend
> >  
> > -from numpy import get_include
> > +from numpy.distutils.core import setup
> 
> Why numpy.xxx?

because it fixes a c++ warning, and includes numpy.get_include()